great_expectations/rule_based_profiler/parameter_builder/numeric_metric_range_multi_batch_parameter_builder.py

```diff
@@ -644,6 +657,19 @@ def _is_metric_values_ndarray_datetime_dtype(
 
         return True
 
+    @staticmethod
+    def _is_metric_values_ndarray_decimal_dtype(
+        metric_values: np.ndarray,
+        metric_value_vector_indices: List[tuple],
+    ) -> bool:
+        metric_value_vector: np.ndarray
+        for metric_value_idx in metric_value_vector_indices:
```

Comment:
@billdirks Perhaps -- I would not outright assume that it cannot be, though I do not want to do it now as part of this PR, because this loop is such that every index points to an `ndarray` (and we are vectorizing that) -- so potentially might require more time to get right.  Thanks.